test/sanity/run-test.sh

```diff
@@ -52,4 +54,4 @@ bin/nfsplugin --endpoint "$endpoint" --nodeid "$nodeid" -v=5 &
 
 echo 'Begin to run sanity test...'
 readonly CSI_SANITY_BIN='csi-test/cmd/csi-sanity/csi-sanity'
-"$CSI_SANITY_BIN" --ginkgo.v --ginkgo.noColor --csi.testvolumeparameters="$(pwd)/test/sanity/params.yaml" --csi.endpoint="$endpoint" --ginkgo.skip="should not fail when requesting to create a volume with already existing name and same capacity|should fail when requesting to create a volume with already existing name and different capacity|should work|should fail when the requested volume does not exist"
```

Comment:
what does `ginkgo.noColor` mean?

Reply:
Removing flag `ginkgo.noColor` will give you the following "colorful" results in the console, which is nice to locate failures.
![image](https://user-images.githubusercontent.com/31323106/99223593-a6fcd880-2799-11eb-8174-614ec3c6e11f.png)